ctwrap/modules/minimal.py

```diff
@@ -1,10 +1,9 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
 """Minimal example for ctwrap compatible simulation module"""
 
-from ruamel import yaml
-import pandas as pd
 import time
+import pandas as pd
```

Comment:
I don't think `pandas` is used anywhere anymore, correct?

Reply:
`Minimal.py`